### ISSUE TITLE
add tidier to broom

### ISSUE THREAD

**Issue by mcjmigdal** [junkka/ehahelper#1]
just stumbled upon your package while thinking about writing `coxme` tidier myself.
Are you considering contributing this part of your code to `broom.mixed`?
It would be much easier to find :)

Cheers MM

**Comment by junkka** [comment]
Yes, I have been following the `broom.mixed` development, but haven't got around contributing to it. This is a good idea, there might be a larger interest in `coxme` tidiers than just for myself.
